app/helpers/users_helper.rb

```diff
@@ -1,2 +1,8 @@
 module UsersHelper
+  def gravatar_for(user, options = { size: 80 })
+    gravatar_id = Digest::MD5.hexdigest(user.email.downcase)
+    size = options[:size]
+    gravatar_url = "https://secure.gravatar.com/avatar/#{gravatar_id}?s=#{size}"
+    image_tag(gravatar_url, alt: user.name, class: 'gravatar')
```

Comment:
__[reek]__
* [FeatureEnvy] refers to user more than self (maybe move it to another class?) [:ref](https://github.com/troessner/reek/blob/master/docs/Feature-Envy.md)